### PR TITLE
Adding folder name

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

File: frontend/encore/versioning.rst [hunk 1]
```diff
@@ -28,7 +28,8 @@ To link to these assets, Encore creates two files ``entrypoints.json`` and
 Loading Assets from ``entrypoints.json`` & ``manifest.json``
 ------------------------------------------------------------
 
-Whenever you run Encore, two configuration files are generated: ``entrypoints.json``
+Whenever you run Encore, two configuration files are generated in your
+``public/build/`` folder: ``entrypoints.json``
```
Comment (BahmanMehrdad) [inline frontend/encore/versioning.rst]: There is a possibility of changing `public/build/` path in `webpack.config.js` file, so I think it is better to use the following text:

`Whenever you run Encore, two configuration files are generated in directory where compiled assets will be stored: ...`

**Review by BahmanMehrdad — CHANGES_REQUESTED** [review]

File: frontend/encore/versioning.rst (hunk 1 above)
Comment (ThomasLandauer) [reply frontend/encore/versioning.rst]: The point of this change it to make it easier for people who *do not know* where stuff gets generated. Those who have changed the setting themselves, will know to what they changed it ;-)

Comment (BahmanMehrdad) [reply frontend/encore/versioning.rst]: I think it is better for the text to be clear and transparent for everyone. Therefore, this text can be placed in such a way that even beginners can use it more easily

`Whenever you run Encore, two configuration files are generated in directory where compiled assets will be stored (By default ``public/build/`` folder) : ...`

Comment (ThomasLandauer) [reply frontend/encore/versioning.rst]: OK. I shortened it a bit. "Output folder" is taken from `.setOutputPath()` (in `webpack.config.js`)

**Review by BahmanMehrdad — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks

**Pull request merged**